pkg/ocm/sca/sca.go

```diff
@@ -125,21 +157,14 @@ func (c *Controller) requestDataAndCheckSecret(ctx context.Context, endpoint str
 		return
 	}
 
-	var ocmRes Response
-	err = json.Unmarshal(data, &ocmRes)
+	err = c.processResponses(ctx, *responses)
 	if err != nil {
-		klog.Errorf("Unable to decode response: %v", err)
-		return
-	}
-	// check & update the secret here
-	err = c.checkSecret(ctx, &ocmRes)
-	if err != nil {
-		klog.Errorf("Error when checking the %s secret: %v", secretName, err)
+		klog.Errorf("Error when checking the sca secret: %v", err)
 		return
 	}
 
-	klog.Infof("%s secret successfully updated", secretName)
-	c.StatusController.UpdateStatus(controllerstatus.Summary{
+	klog.Info("sca secret successfully updated")
```

Comment:
It would be good to keep the `secretName` in the log message.

Reply:
I have moved this log into the `createSecret` and `updateSecret` where I can log the secret name easily